pyproject.toml

```
@@ -147,13 +147,13 @@ crawler = [
   "selenium>=4.11.0"
 ]
 preprocessing = [
-  "nltk",
+  "nltk>=3.9",
```

```suggestion
  "nltk>=3.9.1",
```

I'm not sure, but in the release notes, we refer to 3.9.1.
Right?